src/test/resources/fhir/Bundle/search-bundle.xml

```diff
@@ -1,976 +1,539 @@
-<Bundle xmlns="http://hl7.org/fhir">
+<Bundle xmlns='http://hl7.org/fhir'>
```

Comment:
This seems to be the only xml file in the project that uses single quote characters for attributes, consider reverting to double quote characters to stay consistent.

Reply:
implemented with 178d1fa